rclcpp/include/rclcpp/parameter.hpp

```diff
@@ -224,7 +224,7 @@ class ParameterVariant
     return parameter;
   }
 
-  std::string to_string() const
+  std::string value_to_string() const
```

Comment:
This provides the string for the value of a `ParameterVariant`. The `ParameterValue` is a private member of the `ParameterVariant` an inaccessible to the user. The method could be moved into `ParameterValue` except that it's a message. It could be a free function. But then we'd want to have a method on the `ParameterVariant` to expose it. We can refactor it to use a free function, but I just renamed it to disambiguate it from the new functions which print all the info of the `ParameterVariant` to string not just the value.